client/js/components/drawer.js

```diff
@@ -0,0 +1,25 @@
+class Drawer {
+  constructor(el) {
+    this.el = el;
+    this.trigger = el.getElementsByClassName('js-trigger')[0];
+    this.trigger.addEventListener('click', this.toggleActive.bind(this));
```

Comment:
`addEventListenter` [is perfectly fine though](http://caniuse.com/#feat=addeventlistener). We will start having to decide what we consider a % rate where we don't support the browser.

@gestchild?